Changes incorporating Luis' comments. adding more metadata definitions. (#36)
danwing/metadata: draft-rwbr-sconepro-flow-metadata.md

Video Streaming Metadata:

> The use case requirements for the below table is explained in detail in {{?I-D.-kwbdgrr-tsvwg-net-collab-rqmts}}. The audio is more important than video (importance=high, PT=keep, RU=reliable), video key frames

Reviewer: ```suggestion
The use case requirements for {{table-video-streaming}} is explained in detail in {{?I-D.kwbdgrr-tsvwg-net-collab-rqmts}}. The audio is more important than video (importance=high, PT=keep, RU=reliable), video key frames
```

}
~~~~~

## Interactive Media{#example-interactive-av}

> Based on metadata types listed in the {{?I-D.rwbr-sconepro-flow-metadata}}, the host to network metadata parameters for interactive media type is given below.

Reviewer: ```suggestion
Based on metadata types listed in this document, the host to network metadata parameters for interactive media type is given below.
```

}
~~~~~

## Live Streaming {#example-ls}

> Based on metadata types listed in the {{?I-D.rwbr-sconepro-flow-metadata}}, the host to network metadata parameters for interactive media type is given below.

Reviewer: ```suggestion
Based on metadata types listed in this document, the host to network metadata parameters for interactive media type is given below.
```

|:-----------------:|:----------:|:-----------------:|:--------------------:|
| video key frame   | high        | realtime          | reliable             |
| video delta frame | low        | discard           | unreliable           |
| audio             | low       | discard          | unreliable             |
{: #table-video-livestream title="Example Values for live streaming of video preferred event"}

Metadata for live-streaming that prefers audio over video: (eg. Music Concert)

| Traffic type      | Importance | PacketNature      | PacketType           |
|:-----------------:|:----------:|:-----------------:|:--------------------:|
| video key frame   | low        | realtime          | reliable             |
| video delta frame | low        | discard           | unreliable           |
| audio             | high       | realtime          | reliable             |
{: #table-audio-livestream title="Example Values for live streaming of audio preferred event"}

## Remote Desktop Virtualization {#example-rdt}

Example packet metadata for Desktop Virtualization (like Citrix
Virtual Apps and Desktops - CVAD) application.

Remote Desktop Virtualization Metadata:

> The use case requirements for the below table is explained in detail in {{?I-D.-kwbdgrr-tsvwg-net-collab-rqmts}}. The metadata for the use case is defined as follows:

Reviewer: ```suggestion
The use case requirements for the below table is explained in detail in {{?I-D.kwbdgrr-tsvwg-net-collab-rqmts}}. The metadata for the use case is defined as follows:
```
